apps/wallet/src/screens/Dashboard/Collectible.tsx

```diff
@@ -37,7 +52,7 @@ export const CollectibleFeat = () => {
 	useEffect(() => {
 		// TODO: need to resolve and remove can go back if possible
 		if (!curCollectible && navigationRef.canGoBack()) {
-			// navigateBack();
+			handleBackToHome();
```

Comment:
please make sure that this one is not called twice, the problem will come when you sent an NFT and try to click on the `collection`. Need to recheck this one, it's a separated bug